styles.css

```diff
@@ -93,6 +93,11 @@ ul.navbar-items {
   list-style: none;
 }
 
+.contact-items ul li {
+  margin-bottom: 10px;
+  text-align: center;
+}
```

Comment:
- Hey, let's add some spaces between each icon here on the desktop version and also, update the link to the last icon that is broken to improve our design.

*Figma desktop design*
![image](https://github.com/Xghayor/Portfolio/assets/59397159/99a1673c-6653-4eca-bb38-0d973e674a80)

*Your desktop version design*
![image](https://github.com/Xghayor/Portfolio/assets/59397159/01182d86-1070-45b9-a593-a290c0d64875)


- Let's also align the icons correctly in the mobile version as seen in the image below:

*Figma mobile version*
![image](https://github.com/Xghayor/Portfolio/assets/59397159/5f5cc203-3c6d-4dfa-a2e8-df3baa966be5)

*Your design mobile version*
![image](https://github.com/Xghayor/Portfolio/assets/59397159/47248b37-6331-42d6-996a-5a93c507a88c)

Reply:
@zieeco Hi,
i would like to mention that in my screen it perfectly working, which branch did you downloaded to check?
![1](https://github.com/Xghayor/Portfolio/assets/122298629/e819e66c-98fc-4fa6-8666-6bbcd59214e8)
![2](https://github.com/Xghayor/Portfolio/assets/122298629/4b4b400c-90ee-4b3d-b5e0-bd96a155d4c9)

please have a look on these screenshots, thank you